snapcraft/cli/_errors.py

```diff
@@ -106,7 +106,15 @@ def _submit_trace(exception):
     client = RavenClient(
         'https://b0fef3e0ced2443c92143ae0d038b0a4:'
         'b7c67d7fa4ee46caae12b29a80594c54@sentry.io/277754',
-        transport=RequestsHTTPTransport)
+        transport=RequestsHTTPTransport,
+        # Should Raven automatically log frame stacks (including locals)
+        # for all calls as it would for exceptions.
+        auto_log_stacks=True,
+        # Removes all stacktrace context variables. This will cripple the
+        # functionality of Sentry, as you’ll only get raw tracebacks,
+        # but it will ensure no local scoped information is available to the
+        # server.
+        processors=('raven.processors.RemoveStackLocalsProcessor',))
```

Comment:
I'm not 100% sure what this means given the [docs](https://docs.sentry.io/clients/python/advanced/#python-client-config), can you give me an example?

Reply:
While the `SanitizePasswordsProcessor` mentions that it removes all keys which resemble `password`, `secret`, or `api_key` within stacktrace contexts, HTTP bits (such as cookies, POST data, the querystring, and environment), and extra data. I don't want to accidentally fall in a situation where the regex missed out on macaroons, credit cards (not something we do, today at least). To be fair, knowing the value of a variable in our cases most of the time would not make it hard to figure out why the stacktrace happens in most situations.

Reply:
Ah, okay same page now, thank you!